sway/meson.build

```diff
@@ -210,5 +210,6 @@ executable(
 	include_directories: [sway_inc],
 	dependencies: sway_deps,
 	link_with: [lib_sway_common],
-	install: true
+	install: true,
+	link_args : '-lpthread'
```

Comment:
The correct way to do this in meson is with the 'threads' dependency.

Reply:
Ah thanks.  I've never used meson/ninja before.